gatsby-node.js

```diff
@@ -44,6 +44,29 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
   })
 }
 
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions
+  const typeDefs = `
+    type MarkdownRemark implements Node {
+      frontmatter: Frontmatter
+    }
+    type Frontmatter {
+      hero: Hero
+    }
+    type Hero {
+      cta: Cta
+    }
+    type Cta {
+      icon: Icon
+    }
+    type Icon {
+      alt: String
+      image: File @fileByRelativePath
+    }
```

Comment:
Where does this end up being used? For the CMS?

Reply:
This is so the gatsby query doesn't complain if you decide to not use an icon for the hero CTA. We set it as `required: false` in the cms config, but it isn't actually truly optional without this custom schema because of gatsby images.